### PR TITLE
test: add applitools interactions

### PR DESCRIPTION
**Pull request by MEsteves22**
- Interactions were added to some components in order for applitools to test meaningful baselines: wizard, dialog, drawer, filter group, time picker and date picker.

### CODE REVIEW

**Comment by MEsteves22** [comment]
I'll accept the changes on applitools after the PR is approved.

File: packages/core/src/components/DatePicker/DatePicker.stories.tsx [hunk 1]
```diff
@@ -40,6 +44,15 @@ export const Main: StoryObj<HvDatePickerProps> = {
   argTypes: {
     classes: { control: { disable: true } },
   },
+  parameters: {
+    eyes: {
+      runBefore() {
+        fireEvent.click(screen.getByRole("combobox"));
+
+        return waitFor(() => screen.getByRole("tooltip"));
```
Comment (zettca) [inline packages/core/src/components/DatePicker/DatePicker.stories.tsx:52]: Don't we need to `await` the element to be on the page? 🤔
```suggestion
        await waitFor(() => screen.getByRole("tooltip"));
```

**Review by zettca — APPROVED** [review]
Nice. We really needed this 👍

File: packages/core/src/components/DatePicker/DatePicker.stories.tsx (hunk 1 above)
Comment (MEsteves22) [reply packages/core/src/components/DatePicker/DatePicker.stories.tsx:52]: From what I've read [here](https://www.npmjs.com/package/@applitools/eyes-storybook#runbefore-and-runafter-functions) it's not necessary. The `runBefore` accepts promises and resolves them before taking the screenshot.

**Pull request merged**